### PR TITLE
Fe feat/details css skeleton

### PR DESCRIPTION
**Pull request by Elise-Jones**
# Description
-Created skeleton for details page.

Closes# (issue)

## Type of change
Just styling

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] Testing
- [ ] Refactoring

# Checklist:

- [ x ] My code follows the style guidelines of this project
- [ x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Components/DetailsPage/DetailsPage.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  return (
+    <div className="details-container">
+      <div className="ba">
+      <div className="pic">
+        <img className="details-image" src={logo} alt="Logo" />
+      </div>
+      <div className="ma">
+        <div className="details-holder">
+          <div>
+            <h2>Amanita parcivolvata</h2>
```
Comment (jmartin777) [inline src/Components/DetailsPage/DetailsPage.js]: Great layout - so when we go to start making the call to this we can interpolate into these fields laid out

File: src/Components/DetailsPage/DetailsPage.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
+.details-container {
```
Comment (jmartin777) [inline src/Components/DetailsPage/DetailsPage.css:5]: Great start on the Details Page Styling!

File: src/Components/DetailsPage/DetailsPage.js [hunk 3]
```diff
@@ -25,26 +25,21 @@ const DetailsPage = () => {
             </div>
           </div>
           <div className="info">
-            {/* <h4>Edibility:</h4> */}
```
Comment (jmartin777) [inline src/Components/DetailsPage/DetailsPage.js]: cleanup commits are so satisfying.

**Review by jmartin777 — APPROVED** [review]
Awesome Elise!

File: src/Components/DetailsPage/DetailsPage.css (hunk 2 above)
Comment (andreasorensen) [reply src/Components/DetailsPage/DetailsPage.css:5]: Really great job! I love it!

File: src/Components/DetailsPage/DetailsPage.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            <div className="taxonomy">
+              <h3>Taxonomy:</h3>
+              <p>Genus: "Amanita",</p>
+              <p>Order: "Agraicales",</p>
+              <p>Family: "Amanitaceae",</p>
+              <p>Phylum: "Basidiomycota",</p>
+              <p> Kingdom: "Fungi"</p>
+            </div>
+          </div>
+          <div className="info">
```
Comment (andreasorensen) [inline src/Components/DetailsPage/DetailsPage.js:27]: I think it will be super easy to pass down the props and get this all set up for data! Great job!!

**Review by andreasorensen — APPROVED** [review]

**Pull request merged**